zvariant/src/signature.rs

```diff
@@ -20,29 +22,13 @@ use crate::{signature_parser::SignatureParser, Basic, EncodingFormat, Error, Res
 // breakage.
 //
 // [`bytes::Bytes`]: https://docs.rs/bytes/0.5.6/bytes/struct.Bytes.html
-#[derive(Debug, Hash, Clone, PartialOrd, Ord, Eq)]
+#[derive(Debug, Clone)]
 enum Bytes<'b> {
     Borrowed(&'b [u8]),
     Static(&'static [u8]),
     Owned(Arc<[u8]>),
 }
 
-impl PartialEq for Bytes<'_> {
-    fn eq(&self, other: &Self) -> bool {
-        match (self, other) {
-            (Bytes::Borrowed(b), Bytes::Static(s)) => **b == **s,
-            (Bytes::Static(s), Bytes::Borrowed(b)) => **s == **b,
-            (Bytes::Owned(o), Bytes::Borrowed(b)) => *o.as_ref() == **b,
-            (Bytes::Borrowed(b), Bytes::Owned(o)) => **b == *o.as_ref(),
-            (Bytes::Static(s), Bytes::Owned(o)) => **s == *o.as_ref(),
-            (Bytes::Borrowed(b1), Bytes::Borrowed(b2)) => **b1 == **b2,
-            (Bytes::Owned(b1), Bytes::Owned(b2)) => *b1.as_ref() == *b2.as_ref(),
-            (Bytes::Owned(b1), Bytes::Static(s2)) => *b1.as_ref() == **s2,
-            (Bytes::Static(s1), Bytes::Static(s2)) => **s1 == **s2
-        }
-    }
-}
-
```

Comment:
This is a fixup commit. please squash it into the relevant previous commit and update the log.